procedures/foreman/release.md.erb

```diff
@@ -15,8 +15,8 @@
 - [ ] Update manual if applicable for any additional installation steps
 - Update [release notes section](https://github.com/theforeman/theforeman.org/tree/gh-pages/_includes/manuals/<%= short_version %>/1.2_release_notes.md) in the manual:
   - [ ] Release notes: bullet point list by category of all changes, include link to bug numbers.
-    - [ ] For [theforeman.org](https://github.com/theforeman/theforeman.org): You can auto-generate changes using the [release notes script](https://github.com/theforeman/theforeman.org/blob/gh-pages/scripts/release_notes.rb).
-    - [ ] For [docs.theforeman.org](https://github.com/theforeman/foreman-documentation): You can auto-generate changes using [redmine_release_notes](https://github.com/theforeman/foreman-documentation/blob/<%= short_version %>/guides/doc-Release_notes/redmine_release_notes) script, more details are available in [README](https://github.com/theforeman/foreman-documentation/blob/<%= short_version %>/guides/doc-Release_notes/README.md).
+    - [ ] For [theforeman.org](https://github.com/theforeman/theforeman.org) using the [release notes script](https://github.com/theforeman/theforeman.org/blob/gh-pages/scripts/release_notes.rb): `./scripts/release_notes foreman <%= version %>`
+    - [ ] For [docs.theforeman.org](https://github.com/theforeman/foreman-documentation) using [redmine_release_notes](https://github.com/theforeman/foreman-documentation/blob/<%= short_version %>/guides/doc-Release_notes/redmine_release_notes) script (see [README](https://github.com/theforeman/foreman-documentation/blob/<%= short_version %>/guides/doc-Release_notes/README.md) as well): `./guides/doc-Release_notes/redmine_release_notes foreman <%= version %> > ./guides/doc-Release_Notes/topics/foreman-<%= version %>.adoc`
```

Comment:
This is incomplete. It also needs to be added to `guides/doc-Release_Notes/master.adoc`.